README.md

```diff
@@ -7,6 +7,7 @@ This repository is focused on building and packaging the OpenTelemetry Collector
 
 To generate a tarball file that packages the executable OpenTelemetry binary with the configuration file compatible with Google Cloud Monitoring.
 1. Run `make build-tarball`
+    - To also include JVM, Apache, MySQL and StasD support, run `make build-tarball-exporters CONFIG_FILE=config-mysql_apache_jvm.yaml` instead
```

Comment:
It'd be good to make the `build-tarball-exporters` use that config file by default (but still allow it to be overridable) so this doesn't need to be part of the command users need to run.